analysers/analyser_osmosis_parking_highway.py

```diff
@@ -57,6 +59,40 @@
   highway.id IS NULL
 """
 
+
+sql20 = """
+SELECT
+  parking.id,
+  MAX(parking_way.id),
+  ST_AsText(ST_Centroid(parking.linestring)),
+  -- grab access value of the one belonging to the highlighted way (matching MAX id)
+  (ARRAY_AGG(parking_way.tags->'access' ORDER BY parking_way.id DESC))[1],
+  parking.tags->'access'
+FROM
+  ways AS parking
+  JOIN park_highway AS parking_way ON
+    ST_Intersects(parking.linestring, parking_way.linestring) AND
+    NOT ST_Contains(ST_MakePolygon(parking.linestring), parking_way.linestring)
+WHERE
+  parking.is_polygon AND
+  parking.tags != ''::hstore AND
+  parking.tags?'amenity' AND
+  parking.tags->'amenity' = 'parking' AND
+  (NOT parking.tags?'parking' OR parking.tags->'parking' NOT IN ('street_side', 'lane')) AND
+  (
+    NOT parking.tags?'access' OR
+    (
+      parking.tags->'access' NOT IN ('private', 'permit', 'delivery', 'customers', 'no') AND
+      parking.tags->'access' NOT LIKE '%;%'
+    )
+  )
+GROUP BY
+  parking.id,
+  parking.tags
+HAVING
+  ARRAY_AGG(parking_way.tags->'access') <@ array['private', 'permit', 'delivery', 'customers']
```

Comment:
```suggestion
  array_agg(parking_way.tags->'access') <@ array['private', 'permit', 'delivery', 'customers']
```

It is a function

Reply:
Done :).
I left the ST_* ones intact, as they're used throughout the analysers in various writing methods (`ST_*`, `St_*` and `st_*` all occur).
(`MIN` and `min` are both used in the various analysers; `SUM` function is always used as uppercase, but indeed, array_agg is always used lowercase)